### ISSUE TITLE
[BUG] [v1.2] Display SR Text - When aria attributes only have spaces, the utility fails

### ISSUE THREAD

**Issue by ntibbles** [ntibbles/equa11y#87]
In certain instances, when aria attributes exist but are empty (aria-describedby=" "), the utility fails silently and doesn't display anything. The utility should ignore these instances and display the screen text for other attributes and visual text.

**Comment by ntibbles** [comment]
Fixed. Waiting for build and deployment.
